client/src/App.js

```diff
@@ -1,17 +1,79 @@
 import React from "react";
 import "./App.css";
 import Video from "./Video";
+import AddVideo from "./AddVideo";
 import dataVideos from "./exampleresponse.json";
+import { useState, useEffect } from "react";
 
 function App() {
+  const [dataVideo, setDataVideo] = useState(dataVideos);
+  const [videoID, setVideoID] = useState(null);
+  const deleteVideo = (e) => {
+    let videoID = e.target.value;
+    setVideoID(videoID);
+  };
+  useEffect(() => {
+    let filteredVideos = dataVideo.filter((video) => {
+      return video.id != videoID;
+    });
+    setDataVideo(filteredVideos);
+  }, [videoID]);
+  const [isHidden, setIsHidden] = useState(true);
+  const visibleToolbar = () => {
+    setIsHidden(!isHidden);
+  };
+  const [titleValue, steTitleValue] = useState("");
+  const titleChange = (e) => {
+    console.log(e.target.value);
+    steTitleValue(e.target.value);
+  };
+  const [urlValue, steURLValue] = useState("");
+  const urlChange = (e) => {
+    console.log(e.target.value);
+    steURLValue(e.target.value);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("Sunmited");
+    if (titleValue !== "" && urlValue.includes("youtube.com")) {
```

Comment:
You might want to check this part as when you press into the share button in YouTube, it shares the video in a different format than `youtube.com`, such as `https://youtu.be/ew2vg50IT2c`.

Reply:
That has been sorted. I have included "youtu.be".